team_picker/views/show.ejs

```diff
@@ -0,0 +1,82 @@
+<%- include('./partials/header') %>
+
+<img class="card-img-top logo ml-auto mr-auto" src="<%=cohort.logoUrl%>" alt="Card image cap">
+<h1><%=cohort.name%> </h1>
+<ul>
+  <% for (let member of cohort.members.split(",")) { %>
+    <li class="card-text"><%=member%></li>
+  <% } %>
+</ul>
+<div class="d-flex">
+  <a href="/teams/<%= cohort.id %>/edit" class="btn btn-outline-success btn-sm mr-2">Edit</a>
+  <form action="/teams/<%= cohort.id %>" METHOD="POST">
+    <input type="hidden" name="_method" value="DELETE">
+    <input type="submit" value="Delete" class="btn btn-sm btn-outline-danger">
+  </form>
+</div>
+<br>
+<div>
+  <h4>Team Picker</h3>
+  <form action="/teams" method="POST">
+    <input type="hidden" name="id" value="<%= cohort.id %>">
+    <div class="form-row">
+      <div class="form-group col-md-4">
+        <select class="form-control" name="pickerMethod">
+          <option selected>Choose...</option>
+          <option value="totalTeams">How many teams?</option>
+          <option value="membersPerTeam">How many members per team?</option>
+        </select>
+      </div>
+      <div class="col-md-2">
+        <input type="text" class="form-control" name="quantity" placeholder="Quantity">
+      </div>
+    </div>
+    <button type="submit" class="btn btn-outline-primary">Assign teams</button>
+  </form>
+</div>
+
+<% if (pickerMethod) { %>
```

Comment:
Way too much logic for your template/view. This should all be happening in the [routes](https://github.com/rosadiaz/homeworks/pull/6/files#diff-e79ea032b67d8a0bcdd34e679a2c241fR47) (or probably some external independent class, but you probably haven't gotten around to that quite yet, so for now the routes is probably the next best place). You don't want to put so much of the "business logic" into the view. The route should do all the heavy-lifting to decide based on the picker method and whatnot, and just provide the view with the final teams to output. Then the view's job is just to display what the router set up for it. The syntax here is probably "intentionally" painful, so that you avoid doing it. E.g. having to add `<% } %>` all over the map.

By putting the business logic outside the view (and in the router for now), you can also extract the logic into independent functions that deal with the different options (e.g. one for the "by number of teams" logic, and another one for the "by number of members" logic)